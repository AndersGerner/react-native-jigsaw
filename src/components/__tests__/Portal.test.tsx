import * as React from "react";
import { View } from "react-native";
import renderer from "react-test-renderer";
import Portal from "../Portal/Portal";

it("renders portal", () => {
  const tree = renderer.create(<Portal children={<View />} />).toJSON();

  expect(tree).toMatchSnapshot();
});
