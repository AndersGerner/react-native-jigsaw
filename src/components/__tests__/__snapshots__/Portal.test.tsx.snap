// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders portal 1`] = `null`;
